Automattic/wp-calypso
# Memberships section: product management

This introduces product management in earn / memberships

- Ability to list products
- Editing
- Deleting

This needs D28416-code

## Screenshots

![Zrzut ekranu 2019-05-20 o 12 52 07](https://user-images.githubusercontent.com/3775068/58018277-ac288b00-7b02-11e9-844d-eb065896b023.png)
![Zrzut ekranu 2019-05-20 o 12 52 12](https://user-images.githubusercontent.com/3775068/58018283-ae8ae500-7b02-11e9-9956-e6b58d362b64.png)
![Zrzut ekranu 2019-05-20 o 12 52 17](https://user-images.githubusercontent.com/3775068/58018287-b0ed3f00-7b02-11e9-8301-8849309a01b2.png)
![Zrzut ekranu 2019-05-20 o 12 52 24](https://user-images.githubusercontent.com/3775068/58018290-b2b70280-7b02-11e9-929d-0c8e91c816b2.png)
![Zrzut ekranu 2019-05-20 o 12 52 35](https://user-images.githubusercontent.com/3775068/58018292-b5195c80-7b02-11e9-8b34-d9cad17f1ee6.png)

### Testing instructions

1. Sandbox store, public-api and subscribe.wordpress.com .
2. Apply D28416-code
3. Pick a site that is not on a free plan ( jetpack or wpcom simple )
4. Build this with `ENABLE_FEATURES=memberships npm run start`
5. Go to http://calypso.localhost:3000/earn/memberships/ , pick your site
6. COnnect stripe if not connected,
7. Scroll to bottom of http://calypso.localhost:3000/earn/memberships/  , click "membership amounts"
8. Add an amount
9. Go to wp-admin gutenbrg of your site, search for membership buttom block, pick the amount just created in calypso, should be there.
10. Publish site, click your membership button to see that it works
11. Edit your amount in calypso
12. click the button again, see the change.
13. Delete amount i calypso
14. reload published page, see that button has disappeared.
